src/dev/flang/tools/Fuzion.java

```diff
@@ -724,13 +734,143 @@ else if (args.length >= 1 && args[0].equals("-acemode"))
       {
         return parseArgsAceMode(args);
       }
+    else if (args.length >= 1 && args[0].equals("-coverage"))
+      {
+        return coverage(args);
```

Comment:
Would be good to stick to the existing pattern, i.e., call `parseCoverage(args)` here and move most of the code in `coverage` and the helper methods to a new class `Coverage`. 

I think it would be best to make `coverage` a backend, only then it would work with more complex examples using modules etc. 
Alternatively, if this is just for our internal use and there is no need for this to work with more complex examles, this the option should be called `-Xcoverage`.